build-logic-commons/build-scan/build.gradle.kts

```diff
@@ -20,6 +20,11 @@ plugins {
 
 description = "Provides a plugin that configures build scans in the Gradle build"
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
I suppose if you do this here, then you'd also need to change `gradlebuild.build-logic.groovy-dsl-gradle-plugin.gradle.kts` and `gradlebuild.build-logic.kotlin-dsl-gradle-plugin.gradle.kts` and set `sourceCompatibility` to 11 there as well, right?

Reply:
Yes, they were missing. Fixed here: https://github.com/gradle/gradle/pull/19659